doc/security/specs/identities_and_root_keys/README.md

```diff
@@ -253,39 +243,118 @@ OwnerIntermediateKey =
     <td><strong>Description</strong></td>
   </tr>
   <tr>
-    <td id="owner-root-secret">OwnerRootSecret</td>
+    <td> id="creator-secret"CreatorSeed</td>
     <td>Flash</td>
     <td>
-Used as a diversification constant with acceptable entropy. Provisioned at
-Ownership Transfer time by the Silicon Creator.
+Additional diversification key stored in flash. Provisioned at
+manufacturing time by the Silicon Creator.
 
-The OwnerRootSecret has different visibility options depending on the level of
-isolation provided in hardware:
+Hidden from software once provisioned.
+    </td>
+  </tr>
+  <tr>
+    <td>SoftwareBindingValue</td>
+    <td>SW Register<br>Lockable Input</td>
+    <td>
+For the attestation chain, this register is populated with ROM_EXT measurement.
 
-*   The value should be hidden from software after provisioning.
-*   The value is visible to ROM and ROM Extension, but hidden from all Silicon
-    Owner software. The ROM Extension implements this property.
+For the sealing chain, this register uses a diversification value determined by
+ROM_EXT.
+    </td>
+  </tr>
+</table>
+
+## OwnerIntermediateIdentity
+
+The OwnerIntermediateIdentity is associated with CDI_0 in the DICE attestation
+chain.
+
+```
+OwnerIntermediateIdentitySeed =
+  KM_DERIVE(OwnerIntermediateKey, OTBNDiversificationConstant)
+
+// ASYM_KDF is a KDF function compliant to the Asymmetric Key
+// requirements defined in the Attestation specification document.
+OwnerIntermediateIdentity_Private = ASYM_KDF(OwnerIntermediateIdentitySeed)
+CLEAR_BEFORE_NEXT_BOOT_STAGE(OwnerIntermediateIdentitySeed, OwnerIntermediateIdentity_Private)
+```
+
+<table>
+  <tr>
+    <td><strong>Name</strong></td>
+    <td><strong>Encoding</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td>OwnerIntermediateIdentitySeed</td>
+    <td>SW Register Output</td>
+    <td>Seed used to generate the OwnerIntermediateIdentity asymmetric key.</td>
+  </tr>
+  <tr>
+    <td>OTBNDiversificationConstant</td>
+    <td>Gates</td>
+    <td>
+A constant defined in gates. Used to diversify sideloaded values to OTBN.
+
+Hidden from software.
+    </td>
+  </tr>
+</table>
+
+The `OwnerIntermediateIdentitySeed` and the private portion of the
+OwnerIntermediateIdentity shall be cleared before the bootloader (BL0) hands
+over execution to the kernel.
+
+## OwnerKey
+
+The `OwnerKey` is the key diversified with `OwnerSeed` and it is associated
+with the CDI_1 certificate.
```

Comment:
maybe you should say it also provides a cryptographic link to the Owner FW image and Owner Configuration Block?